JSDemos/Demos/Form/Validation/description.md

```diff
@@ -1,10 +1,27 @@
-﻿This demo shows how to&nbsp;validate the [Form][0] editors. [Validation rules][1] are specified for each form item using the **validationRules** property. All editors on&nbsp;the form are combined into a&nbsp;validation group defined by&nbsp;the form&rsquo;s [validationGroup][2] property. 
+﻿This demo shows how to validate the Form editors in simple items. To apply validation rules to an item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/) array. Specify [type](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#type) and other properties for each rule. 
 
+To submit a Form, add a [Button Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/) and set its [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property to **true**. When users click the button, the Form validates all editors that belong to the same group. In this demo, all editors are wrapped into the `customerData` [validationGroup](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#validationGroup). The Form can be submitted to a server only if input validation is successful. Set the [showValidationSummary](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#showValidationSummary) property to **true** to display all validation errors at the bottom of the Form. 
 
-Note that the _&laquo;Register&raquo;_ button here does not implement a&nbsp;usual [onClick][3] event handler. Instead, it&nbsp;has the [useSubmitBehavior][4] property set to _true_. This setting tells the button to&nbsp;validate and submit the HTML form in&nbsp;which it&nbsp;is&nbsp;nested, with no&nbsp;further configuration required.
+The **useSubmitBehavior** property requires that you wrap the dxForm in the HTML `<form>` element. You should also set the [preventDefault](/Documentation/ApiReference/Common/Object_Structures/dxEvent/Methods/#preventDefault) property to **true** to override the HTML form submit event. 
```

Comment:
The information about wrapping our Form in the HTML form moved to the previous paragraph. Regarding preventDefault, it is only needed in this particular demo to stop the form from actually being submitted. I think we can drop this information altogether.

```suggestion
```